### PR TITLE
Update --help text for --embed-file/--preload-file. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: docs/emcc.txt [hunk 1]
```diff
@@ -322,9 +322,8 @@ Options that are modified or new in *emcc* are listed below:
 
    Note:
 
-     Embedding files is much less efficient than preloading them. You
-     should only use it for small files, in small numbers. Instead use
-     "--preload-file", which emits efficient binary data.
+     Embedding files is genearlly more efficient than preloading them
```
Comment (dschuff) [inline docs/emcc.txt:325]:
```suggestion
     Embedding files is generally more efficient than preloading them
```

File: docs/emcc.txt [hunk 2]
```diff
@@ -322,9 +322,8 @@ Options that are modified or new in *emcc* are listed below:
...
+     can avoid copying/duplicating the file data.
```
Comment (dschuff) [inline docs/emcc.txt:326]:
```suggestion
     as it avoids copying/duplicating the file data.
```

Comment (dschuff) [reply docs/emcc.txt:326]: Can you expand this a bit? Copying/duplicating the file data where? Does it both duplicate (at compile time) and copy (at runtime)?

Comment (sbc100) [reply docs/emcc.txt:326]: With preloading the file data is download by JS and copied in the linear memory, so it is both copied and (until GC runs anyway) duplicated in memory (1 copy in native, 1 in JS).

Comment (sbc100) [reply docs/emcc.txt:326]: Happy to rephrase but I think maybe keeping it simple is good.  Should I just say "copying"?

Comment (sbc100) [reply docs/emcc.txt:326]: I re-phased it.

**Review by dschuff — APPROVED** [review]

**Pull request merged**